templates/hazelcast5-tls-ec2/inventory_plan.yaml

```diff
@@ -0,0 +1,42 @@
+provisioner: terraform
+terraform_plan: aws
+# Used for naming resources; give it some unique name specific to a set of benchmarks
+basename: <id>-<rnd:5>
+# Enter something here that identifies you.
+owner: <id>
+region: eu-central-1
+availability_zone: eu-central-1a
+#Engineering account VPC
+vpc_id: vpc-094e507d79a701227 
+#Engineering account igw
+internet_gateway_id: igw-03696757cca398137
+#Change the '20' to a different octet to prevent running into conflicts.
+cidr_block: 10.0.20.0/24
+#Change team information
+team: Cloud
+type: Benchmarking
+
+keypair:
+    public_key: key.pub
+    private_key: key
+
+nodes:
+    count: 1
+    instance_type: c5.9xlarge
+    ami: ami-04e601abe3e1a910f
+    user: ubuntu
+    tenancy: null
+    
+loadgenerators:
+    count: 1
+    instance_type: c5.9xlarge
```

Comment:
Based on https://hazelcast.atlassian.net/wiki/spaces/TES/pages/5867274253/Visa+Workload+Validation+Benchmarking+Client+Count+-+Test+Document#Client-Machine-Specification we plan to use c5a.8xlarge for client